diff --git a/node_modules/firebase-tools/lib/emulator/functionsEmulator.js b/node_modules/firebase-tools/lib/emulator/functionsEmulator.js
index 0000000..1111111 100644
--- a/node_modules/firebase-tools/lib/emulator/functionsEmulator.js
+++ b/node_modules/firebase-tools/lib/emulator/functionsEmulator.js
@@ -266,7 +266,6 @@ class FunctionsEmulator {
             const watcher = chokidar.watch(backend.functionsDir, {
                 ignored: [
                     /.+?[\\\/]node_modules[\\\/].+?/,
-                    /(^|[\/\\])\../,
                     /.+\.log/,
                     /.+?[\\\/]venv[\\\/].+?/,
                     ...((_b = (_a = backend.ignore) === null || _a === void 0 ? void 0 : _a.map((i) => `**/${i}`)) !== null && _b !== void 0 ? _b : []),
